### DISCUSSION TITLE
I want to use SOX on Linux, I have SOX installed, but I don't know how to play sounds, I need a demo

### DISCUSSION THREAD

**Discussion by euphoriaer** [igece/SoxSharp#23] [Q&A] [answered]
I want to use SOX on Linux, I have SOX installed, but I don't know how to play sounds, I need a demo，What should I do?

using (Sox sox = new Sox("sox.exe"))
I'm going to make a mistake：
Unhandled exception. System.IO.FileNotFoundException: SoX executable not found

**Comment by igece** [comment, marked as answer]
When you create a Sox instane, you need to provide the full path where the sox executable is (unless it is in the same directory as your application). I don't know where sox is usually installed in Linux, but should be something similar to: `using (var sox = new Sox("/usr/bin/sox/sox")`
